apps/website/next-sitemap.config.cjs

```diff
@@ -0,0 +1,36 @@
+/** @type {import('next-sitemap').IConfig} */
+module.exports = {
+  // If there is a NEXT_PUBLIC_VERCEL_URL set, use that like NextAuth.js does
+  siteUrl: process.env.NEXT_PUBLIC_VERCEL_URL
+    ? `https://${process.env.NEXT_PUBLIC_VERCEL_URL}`
+    : process.env.NEXT_PUBLIC_BASE_URL,
+  // Create a single sitemap, and don't list some routes in it
+  generateIndexSitemap: false,
+  exclude: [
+    "/api",
+    "/api/*",
+    "/auth",
+    "/auth/*",
+    "/admin",
+    "/admin/*",
+    "/wip",
+    "/wip/*",
+    "/giveaways",
+    "/giveaways/*",
+    "/show-and-tell/*",
+  ],
+  // TODO: Transform to apply proper priority to pages
+  // Create a robots txt file, and disallow indexing of some routes
+  generateRobotsTxt: true,
+  robotsTxtOptions: {
+    policies: [
+      {
+        userAgent: "*",
+        disallow:
+          process.env.NEXT_PUBLIC_NOINDEX === "true"
+            ? ["/"]
+            : ["/api/", "/auth/", "/admin/", "/wip/", "/giveaways/"],
```

Comment:
can we add `/privacy-policy` here as well? there are some people searching for "bad" texts policies on search engines trying to sue people (at least here in germany).

Reply:
Sure